.circleci/config.yml

```
@@ -1,10 +1,10 @@
 version: 2.1
 orbs:
-  codecov: codecov/codecov@1.0.5
+  codecov: codecov/codecov@4.1.0
```

```suggestion
  codecov: codecov/codecov@1.0.5
```

Going to try the old version to see if it's a Docker version issue somehow.